livekit/src/room/participant/mod.rs

```diff
@@ -44,6 +44,7 @@ pub enum ParticipantKind {
     Egress,
     Sip,
     Agent,
+    Connector,
 }
 
```

Comment:
Along with the `livekit-protocol` changes, I had to add a new corresponding `Connector` value to this enum to get it to compile again. I don't know what this pertains to / why this was added, so I wanted to call it out just to make sure somebody else more familiar with the state of this repository can verify I did the right thing here.

Reply:
Ugh, yea, I didn't do something right here and this is what is causing the ci build to break... 😞